scripts/release-controller.sh

```diff
@@ -97,34 +94,17 @@ IMAGE_TAG=${IMAGE_TAG:-$VERSION}
 CHART_REPOSITORY=${CHART_REPOSITORY:-$DEFAULT_CHART_REPOSITORY}
 CHART_REGISTRY=${CHART_REGISTRY:-$DEFAULT_CHART_REGISTRY}
 
-ECR_PUBLISH_ROLE_ARN=${ECR_PUBLISH_ROLE_ARN:-"undefined"}
-
-if [[ "$ECR_PUBLISH_ROLE_ARN" = undefined ]]; then
-    error_msg "Please set the ECR_PUBLISH_ROLE_ARN environment variable to the IAM Role ARN"
-    error_msg  "that has permissions to upload artifacts to the ECR Public registry for"
-    error_msg "$IMAGE_REPOSITORY"
-    exit 2
-fi
-
 echo "VERSION is $VERSION"
 
-# TODO(jaypipes): If we move to Prow setup like ACK, uncomment this. For now,
-# just pass in ECR_PUBLISH_ROLE_ARN manually.
-# ASSUME_EXIT_VALUE=0
-# ECR_PUBLISH_ROLE_ARN=$(aws ssm get-parameter --name /ack/prow/cd/public_ecr/publish_role --query Parameter.Value --output text 2>/dev/null) || ASSUME_EXIT_VALUE=$?
-# if [ "$ASSUME_EXIT_VALUE" -ne 0 ]; then
-#   echo "release-controller.sh] [SETUP] Could not find the iam role to publish images to public ecr repository"
-# exit 1
-# fi
-export ECR_PUBLISH_ROLE_ARN
-echo "release-controller.sh] [SETUP] exported ECR_PUBLISH_ROLE_ARN"
-
+#  This Role has permission to publish to the image and Helm chart repositories on ECR public registry in 606627242267 account.
+#  Make sure the aws principal you use to run this script has permission to assume this role
+ECR_PUBLISH_ROLE_ARN=arn:aws:iam::606627242267:role/ECRPublisher
```

Comment:
what is this account? 606627242267

Reply:
<img width="1580" alt="image" src="https://github.com/aws/aws-application-networking-k8s/assets/32318664/e55e4f49-e864-4d34-9ed3-a887b49da6b8">

The controller ECR repos reside in the 606627242267 account, the release-controller.sh script need to assume this role to get permission to upload the container image and helm chart to the repos.


The image-builder ec2 instance in the account 281979210680 account use the same role:

<img width="794" alt="image" src="https://github.com/aws/aws-application-networking-k8s/assets/32318664/fbf5e566-e10b-434f-8280-7c6fb6c6eaf7">